orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/poller/SaltJobIdTracker.java

```diff
@@ -139,6 +140,9 @@ private void checkJobFinishedWithSuccess() throws CloudbreakOrchestratorFailedEx
                 LOGGER.debug("The job (jid: {}) completed successfully on every node.", jobId);
                 saltJobRunner.setJobState(JobState.FINISHED);
             }
+        } catch (SaltEmptyResponseException e) {
+            LOGGER.debug("Jid info is emtpy (jid: {}), this usually occurs due to salt not working properly", jobId, e);
```

Comment:
```suggestion
            LOGGER.debug("Jid info is empty (jid: {}), this usually occurs due to salt not working properly", jobId, e);
```

Reply:
@sxxgrc OMG thank you so much!! Updated!